amundsen_application/api/metadata/v0.py

```diff
@@ -389,14 +389,16 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
 def get_user() -> Response:
 
     @action_logging
-    def _log_get_user(*, user_id: str) -> None:
+    def _log_get_user(*, user_id: str, index: int, source: str) -> None:
```

Comment:
@jinhyukchang Are there any consequences of logging a value of `None`? I've changed the default value for `index` and `source` to `None` on line 83-84.

Reply:
I don't think so as it will serialized as a python `dict`. What was the default value before?

Nit: btw, I think the type of index `int` should be `Optional[int]` as it's now nullable.